src/Behat/Testwork/ServiceContainer/Configuration/ConfigurationTree.php

```diff
@@ -30,13 +30,12 @@ final class ConfigurationTree
      */
     public function getConfigTree(array $extensions)
     {
-        $tree = new TreeBuilder();
-        $root = $tree->root('testwork');
+        $root = new TreeBuilder('testwork');
```

Comment:
this is **not** the root. Please keep the variable name `$tree` for the TreeBuilder`.